apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/PathAssertionsSpec.kt

```diff
@@ -21,6 +23,7 @@ class PathAssertionsSpec : ch.tutteli.atrium.specs.integration.PathAssertionsSpe
     "toBe ${executable::class.simpleName}" to Companion::isExecutable,
     "toBe ${aRegularFile::class.simpleName}" to Companion::isRegularFile,
     "toBe ${aDirectory::class.simpleName}" to Companion::isDirectory,
+    fun2(Expect<Path>::containss),
```

Comment:
```suggestion
  "contains not yet implemented in this API" to Companion::contains
```

And then in the companion object you use:
```
fun contains(expect: Expect<Path>, path: String, vararg otherPaths: String) = ...
``` 
and just copy the logic from api-fluent